raft.c

```diff
@@ -1618,7 +1626,13 @@ static void handleCfgChange(RedisRaftCtx *rr, RaftReq *req)
             RedisModule_ReplyWithSimpleString(req->ctx, rr->snapshot_info.dbid);
             break;
         case RR_CFGCHANGE_REMOVENODE:
-            // block until removed, so don't reply here
```

Comment:
i'm wondering if the original comment should remain in some shape or form to allow future readers to understand why there is no reply here.

Reply:
Didn't realize it, reverted